src/wbi/remote/remote.py

```diff
@@ -64,9 +69,12 @@ def submit(
     return job_id, remote_temp_stdout_path
 
 
-def parse_remote_stdout(client, remote_temp_stdout_path):
+def parse_remote_stdout(client, remote_temp_stdout_path=None):
     try:
-        cmd = f"cat {remote_temp_stdout_path}"
+        if remote_temp_stdout_path:
+            cmd = f"cat {remote_temp_stdout_path}"
+        else:
+            cmd = 'bash -c "echo Hello from Head Node: $(hostname)"'
```

Comment:
The `cmd` we want to run in non-slurm mode is simply the script that was rendered on the remote server (the same one that is used in the sbatch case), after making it executable programmatically (`chmod +x`).

Reply:
Added command to make the remote script executable, and redirected the output to `remote_temp_stdout_path` so that both slurm result and non-slurm result can be parsed in the same manner. 

The updated code looks like: 
```
chmod_command = f"chmod +x {remote_temp_script_path}"
client.exec_command(chmod_command)

cmd = f"bash {remote_temp_script_path} >> {remote_temp_stdout_path}"
client.exec_command(cmd)
```